src/schemas/profile.json

```diff
@@ -51,6 +51,24 @@
           "title": "farcaster",
           "pattern": "^[a-z0-9-]*$",
           "maxLength": 17
+        },
+        "emailSubscription": {
+          "type": "object",
+          "properties": {
+            "email": {
```

Comment:
Maybe emailHash?

Reply:
We need to pass whole readable email, to save in the database.

After, it will not be needed anymore, and `email` will be omitted. To unsubscribe, we pass `email: null`

Reply:
Added a new `Subscription` schema. Sequencer will not create IPFS for this schema